cloud/workspace/workspace.go

```diff
@@ -1,18 +1,30 @@
 package workspace
 
 import (
+	httpContext "context"
+	"fmt"
 	"io"
+	"os"
 	"strconv"
 
 	"github.com/pkg/errors"
 
 	astro "github.com/astronomer/astro-cli/astro-client"
+	astrocore "github.com/astronomer/astro-cli/astro-client-core"
+	"github.com/astronomer/astro-cli/cloud/user"
 	"github.com/astronomer/astro-cli/config"
+	"github.com/astronomer/astro-cli/context"
 	"github.com/astronomer/astro-cli/pkg/input"
 	"github.com/astronomer/astro-cli/pkg/printutil"
 )
 
-var errInvalidWorkspaceKey = errors.New("invalid workspace selection")
+var (
+	errInvalidWorkspaceKey = errors.New("invalid workspace selection")
```

Comment:
is this used? looks to be same as `ErrInvalidWorkspaceKey`

Reply:
yeah ErrInvalidWorkspaceKey is not needed